File: C/colleen/colleen.c
#include <stdio.h>

void	dummy_function(void) {
}
//Ceci est le commentaire en dehors de la fonction
int	main(void) {
char *self = "#include <stdio.h>%c%cvoid	dummy_function(void) {%c}%c%c%cCeci est le commentaire en dehors de la fonction%cint	main(void) {%cchar *self = %c%s%c;%c	printf(self, 10, 10, 10, 10, 47, 47, 10, 10, 34, self, 34, 10, 10, 47, 47, 10, 10);%c%c%cCeci est le commentaire dans la fonction%cdummy_function();%c}";
	printf(self, 10, 10, 10, 10, 47, 47, 10, 10, 34, self, 34, 10, 10, 47, 47, 10, 10);
//Ceci est le commentaire dans la fonction
dummy_function();
}

File: RUST/Colleen/Colleen.rs
fn dummy_function() {}

// Ceci est mon premier commentaire en RUST
fn main() {
//Ceci est mon deuxieme commentaire en Rust
	let code = "fn dummy_function() {}\n\n// Ceci est mon premier commentaire en RUST\n//Ceci est mon deuxieme commentaire en Rust
\nfn main() {\n\tlet code = {:?};\n\tprint!(\"{}\", code.replacen(\"{:?}\", &format!(\"{:?}\", code), 1));\n\tdummy_function();\n}";
	print!("{}", code.replacen("{:?}", &format!("{:?}", code), 1));
	dummy_function();
}